config/environments/development.rb

```diff
@@ -51,4 +51,6 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+
+  config.local_uri_host = 'localhost:3000'
```

Comment:
What if I don't run my local webserver on `localhost:3000`?  (I run local Hyacinth on 3000 and would point to local 3001 for local URI Service).  My point is that maybe that's actually a good case for encoding this value in a file that isn't committed to the repo.  I know we talked about this a while back and I didn't have a good enough justification for a separate file, but I feel like I have a concrete use case now.

Reply:
okay, we can move it to `uri_service.yml`. It also didn't really make sense at the time because it would be the only thing in that config file, but now we have other candidates (api keys, etc).